packages/circuit-ui/components/AvatarInput/AvatarInput.tsx

```diff
@@ -0,0 +1,317 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import {
+  useState,
+  useRef,
+  InputHTMLAttributes,
+  ChangeEvent,
+  Fragment,
+} from 'react';
+import { css, jsx } from '@emotion/core';
+import { Bin } from '@sumup/icons';
+
+import Avatar from '../Avatar';
+import Label from '../Label';
+import IconButton from '../IconButton';
+import Spinner from '../Spinner';
+import ValidationHint from '../ValidationHint';
+import styled, { StyleProps } from '../../styles/styled';
+import { uniqueId } from '../../util/id';
+import { focusOutline, hideVisually } from '../../styles/style-mixins';
+
+export interface AvatarInputProps
+  extends Omit<InputHTMLAttributes<HTMLInputElement>, 'size' | 'onChange'> {
+  /**
+   * A clear and concise description of the image input's purpose.
+   */
+  label: string;
+  /**
+   * A unique identifier for the input element. If not defined, a generated id is used.
+   */
+  id?: string;
+  /**
+   * An existing image URL to be displayed in the image input.
+   */
+  imageUrl?: string;
```

Comment:
I wonder if we should call this `src` as well to match the Avatar component?

Reply:
Yes! This would make sense 🙂